android/src/com/mapswithme/maps/bookmarks/BookmarksListFragment.java

```diff
@@ -812,4 +816,19 @@ private void updateLoadingPlaceholder(@NonNull View root, boolean isShowLoadingP
     UiUtils.showIf(!isShowLoadingPlaceholder, root, R.id.show_on_map_fab);
     UiUtils.showIf(isShowLoadingPlaceholder, loadingPlaceholder);
   }
+
+  @Override
+  public ArrayList<MenuBottomSheetItem> getMenuBottomSheetItems(String id)
+  {
+    if (id.equals(BOOKMARKS_MENU_ID))
```

Comment:
Wouldn't it be better/safer (null?) to use a switch + enum?

Reply:
Handling string enums in Java is not the easiest and creates a lot of boiler plate code. I felt used constants was better for those 2 simple cases.